connectors/bigtable/flink-connector-gcp-bigtable/src/test/java/com/google/flink/connector/gcp/bigtable/BigtableSinkTest.java

```diff
@@ -33,10 +33,6 @@
  */
 public class BigtableSinkTest {
 
-    /**
-     * Tests the {@link BigtableSink} builder and verifies that the configured parameters are
-     * correctly assigned to the sink object.
-     */
     @Test
     public void testBigtableSink() {
```

Comment:
since we removed the comment, the test name should be more descriptive. ie what's the conditions for the test and what is the expected behavior? ref: https://engdoc.corp.google.com/eng/doc/devguide/testing/unit/best-practices.md?cl=head&polyglot=go#naming

Reply:
testBigtableSinkBuilder?